.github/workflows/reusable-build-extensions-on-linux.yml

```
@@ -50,14 +50,14 @@ jobs:
       output_dir: build/plugins/${{ matrix.dir }}
     steps:
       - name: Harden the runner (Audit all outbound calls)
-        uses: step-security/harden-runner@20cf305ff2072d973412fa9b1e3a4f227bda3c76 # v2.14.0
+        uses: step-security/harden-runner@e3f713f2d8f53843e71c69a996d56f51aa9adfb9 # v2.14.1
         with:
           egress-policy: audit
 
       - id: var
         run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:1:110: Double quote to prevent globbing and word splitting

(shellcheck)